src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1283,7 +1283,31 @@ void PolicyHandler::OnSnapshotCreated(const BinaryMessage& pt_string) {
     LOG4CXX_ERROR(logger_, "Service URLs are empty! NOT sending PT to mobile!");
     return;
   }
-  SendMessageToSDK(pt_string, urls.front().url.front());
+
+  static size_t current_app = 0;
```

Comment:
@tkireva could you please give some explanation for next code. It's not very clear what do you want to do. Why variables are `static`?